packages/node_modules/@ciscospark/react-component-utils/src/adaptiveCardUtil.js

```diff
@@ -2,9 +2,31 @@ import * as AdaptiveCard from 'adaptivecards';
 import MDTextInput from '@ciscospark/react-component-md-text-input';
 import MDToggleInput from '@ciscospark/react-component-md-toggle-input';
 import MDChoiceSetInput from '@ciscospark/react-component-md-choiceset-input';
+import {parseMarkdown} from '@ciscospark/redux-module-activity';
 
 import {CARD_ATTACHMENT_TYPE, TEXT_INPUT_ELEMENT, TOGGLE_INPUT_ELEMENT, CHOICE_SET_INPUT_ELEMENT} from './constants';
 
+/* eslint no-param-reassign: 2 */
```

Comment:
esLinit is disabled at a functional level because it's not allowing to assign values to params

Reply:
There's no need to disable this rule. You can create a new object instead of modifying the param object.

Reply:
created a new object and assigned a new object to the result again. Removed  esLint rule